src/entry.js

```diff
@@ -1,3 +1,5 @@
+require('./assets/stylesheets/base.scss');
```

Comment:
to your question about whether or not this is right, the way to verify would be to build the project `npm run build:publish` and see what's in the `lib/` directory.

i think this will probably produce this file as an artifact, but am i going to have to include 2 separate stylesheets in order to use this in another project? that would be my only question